README.md

```diff
@@ -1,5 +1,5 @@
-1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_parse_list_DOM/)
+1. Replace `DmytroPAvlichenko` with your Github username in the link
+    - [DEMO LINK](https://DmytroPAvlichenko.github.io/js_task_parse_list_DOM/)
```

Comment:
Add this demo link to the PR description 
<img width="926" alt="image" src="https://github.com/user-attachments/assets/ca65d28f-74b4-420f-a8fa-a2e77450cdf7" />